internal/runner/run_result.go

```diff
@@ -149,11 +156,12 @@ func (r *RunResult) Statistics() RunStatistics {
 	}
 
 	return RunStatistics{
-		Total:            len(r.tests),
+		Total:            len(r.tests) + len(r.errors),
```

Comment:
I'm not sure if we should add the error count to the total because error doesn't necessarily have 1to1 relationship with a test. But, it will be confusing if we exclude it from the total because the error is displayed in the same table. Any thoughts? @wooly @niceking

Reply:
I feel like if the test throws an exception then the test runner should pass this straight up to the orchestrator and it should fail CI immediately. Is this what happens if you just run straight rspec with a test that throws an exception, or does it keep going until it gets to the end?

Reply:
> I feel like if the test throws an exception then the test runner should pass this straight up to the orchestrator and it should fail CI immediately. Is this what happens if you just run straight rspec with a test that throws an exception, or does it keep going until it gets to the end?

It depends on the runner. RSpec and Jest will keep running the other tests, but Playwright will immediately exit with error.

Reply:
Ah all good then. I think it's reasonable to add an entry to the table for the count of errors detected.

> because error doesn't necessarily have 1to1 relationship with a test

Do you have an example of this?

Reply:
To clarify, any error that prevents the runner from “completing” the tests is immediately terminated by bktec. By “completing” the tests, I mean that the runner has finished all the tests, regardless of their status or any errors outside of the expected ones (e.g., runtime errors).

Reply:
> > because error doesn't necessarily have 1to1 relationship with a test
> 
> Do you have an example of this?
> 

The following test file will be reported as having 1 error, and 0 examples.
```
RSpec.describe "bad syntax" do

  before do
    if "this block is missing an end"
  end

  it "passes" do
    expect(true).to be_truthy
  end

  it "fails" do
    expect(true).to be_falsey
  end
end
```

Reply:
Ah I see. In that case then I wouldn't allow the error count to contribute towards the 'total', since it's not directly related to the count of tests.

Reply:
That's what I thought initially. What about we show the total, then the error, something like this
```
+++ ========== Buildkite Test Engine Report  ==========
┌─────────────┬─────────────┬───┐
│ Passed      │ first run   │ 1 │
│             ├─────────────┼───┤
│             │ on retry    │ 0 │
├─────────────┼─────────────┼───┤
│ Muted       │ passed      │ 0 │
│             ├─────────────┼───┤
│             │ failed      │ 0 │
├─────────────┼─────────────┼───┤
│ Failed      │             │ 1 │
├─────────────┴─────────────┼───┤
│                Total test │ 2 │
├───────────────────────────┼───┤
│               Total error │ 1 │
└───────────────────────────┴───┘
```

Reply:
We decided to remove the error count from the table.